pkg/kubectl/resource/builder.go

```diff
@@ -274,12 +275,48 @@ func (b *Builder) SelectAllParam(selectAll bool) *Builder {
 	return b
 }
 
+func hasNamesArg(args []string) bool {
+	if len(args) > 1 {
+		return true
+	}
+	newArgs := []string{}
+	for _, s := range args {
+		newArgs = append(newArgs, strings.Split(s, ",")...)
+	}
+	args = newArgs
+	for _, s := range args {
+		if !strings.Contains(s, "/") {
+			return false
+		}
+		tuple := strings.Split(s, "/")
+		if len(tuple) != 3 && isGroup(tuple[0]) {
```

Comment:
I discussed with @lavalamp earlier and we think that presumably we won't have duplications on group and resource names. What do you think?